app/models/picture.rb

```diff
@@ -5,7 +5,7 @@ class Picture < ApplicationRecord
             :inclusion => { :in => %w(png jpg svg), :message => 'must be a png, jpg, or svg' },
             :if        => :extension
 
-  virtual_has_one :image_href, :class_name => "String"
+  virtual_has_one :image_href, :class_name => "String", :uses => :binary_blob
```

Comment:
? @Fryguy what's this do? This wasn't here before

Reply:
Ah yeah, I noticed that while `/api/services?expand=resources` didn't need that the actual call made by the SUI is `http://localhost:3000/api/services?expand=resources&attributes=picture%2Cpicture.image_href%2Cevm_owner.name%2Cv_total_vms&filter\[\]=service_id%3Dnil`.  I was trying to optimize that picture reference, but didn't complete it and this was left in.

Technically it's correct, and won't hurt anything to leave in, but I can remove if you think it's better.

Reply:
Just checking; Totally up to you, if you feel confident with it leave it in. 👍